api/scripts/neutralize-answers-after-invalid-challenges.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+require('dotenv').config({ path: `${__dirname}/../.env` });
+const _ = require('lodash');
+const bluebird = require('bluebird');
+
+const { knex } = require('../db/knex-database-connection');
+const logger = require('../lib/infrastructure/logger');
+
+const challengeRepository = require('../lib/infrastructure/repositories/challenge-repository');
+const certificationAssessmentRepository = require('../lib/infrastructure/repositories/certification-assessment-repository');
+const usecases = require('../lib/domain/usecases/index');
+const events = require('../lib/domain/events');
+
+const extractLengthyChallenges = async () => {
+  const challenges = await challengeRepository.findValidated();
+  const lengthyChallenges = _.remove(challenges, function (challenge) {
+    return challenge.validator.solution.value.length >= 500;
+  });
+
+  logger.info(`${lengthyChallenges.length} challenges found`);
+  logger.debug(lengthyChallenges);
```

Comment:
A mettre apres la ligne 25
```suggestion
  logger.debug('Lengthy Challenges:', lengthyChallenges});
```

Reply:
Bien vu, corrigé !